agents/human_agent.py

```diff
@@ -8,8 +9,31 @@ class HumanAgent(Agent):
     the user choose the action.
     TODO - implement everything!
     """
+
     def __init__(self, name):
         super(HumanAgent, self).__init__(name)
 
     def choose_action(self, env):
-        pass
\ No newline at end of file
+        # create list of available moves
+        available_moves = env.available_moves(self)
+
+        # check if the list are not empty
+        if available_moves:
+
+            # action of the player
+            while True:
```

Comment:
This shouldn't run forever...  
By `choose_action` I meant that the player chooses a single action for the current state of the environment, and returns the action (without applying the action on the environment directly).  
Please return the action the human player chooses.

Reply:
This cycle will not work forever:
If the player chooses an available move, then the "return" statement should stop the loop. if not, then after pressing “q” our entire program will stop completely ...

Reply:
You're right, but you might want to add a condition to the loop, just in case.
For example, while not env.is_terminal_state():

Reply:
Lev?

Reply:
Ok, right now it looks fine